firmware/src/configuration.cpp

```diff
@@ -107,42 +107,41 @@ bool Configuration::saveToDisk()
     pb_buffer_.version = PERSISTENT_CONFIGURATION_VERSION;
     if (!pb_encode(&stream, PB_PersistentConfiguration_fields, &pb_buffer_))
     {
-        char buf[200];
-        snprintf(buf, sizeof(buf), "Encoding failed: %s", PB_GET_ERROR(&stream));
-        log(buf);
+        char buf_[200];
+        snprintf(buf_, sizeof(buf_), "Encoding failed: %s", PB_GET_ERROR(&stream));
+        LOGE(buf_);
         return false;
     }
-    ESP_LOGD("1", "save to disk");
+    LOGD("1 Save to disk");
```

Comment:
can you remove those 1-5 save to disk, they were for debugging and we can drop them at this point